### ISSUE TITLE
Issues running dev script

### ISSUE THREAD

**Issue by NormySan** [express-labs/pure-react-carousel#45]
Hi,

I just forked the project to add support for mouse dragging but when I start the dev script i get a console error saying "React is not defined".

I'm not sure how to solve this since I've never set up a dev script with rollup before.

This is the full stack trace.

```
ButtonFirst.jsx:6 Uncaught ReferenceError: React is not defined
    at ButtonFirst.jsx:6
    at getOwnPropertySymbols (index.umd.js:5)
    at index.umd.js:6
(anonymous) @ ButtonFirst.jsx:6
getOwnPropertySymbols @ index.umd.js:5
(anonymous) @ index.umd.js:6
```

**Comment by mrbinky3000** [comment]
Back from vacation.  Fix should be out today.

**Comment by mrbinky3000** [comment]
Fixed in v1.10.0

**Comment by NormySan** [comment]
Great, thanks a lot! 👍
